Add chunking documentation (#424)
spiceai/docs: spiceaidocs/docs/api/http/search.md

pagination_next: null
---

> Performs a basic vector similarity search from one or more dataset(s).

Reviewer: ```suggestion
Performs a basic vector similarity search across one or more datasets.
```

Request Body

>  - `datasets` (array of strings): Dataset component names to perform similarity search against. Each dataset is expected to have one and only one column augmented with an embedding.

Reviewer: ```suggestion
 - `datasets` (array of strings): Names of the dataset components to perform the similarity search on. Each dataset must have exactly one column augmented with an embedding.
```

  "duration_ms": 42,
}
```

> ### Chunking

Reviewer: I think this should be in a Search section, and referenced from the API section?
